### PR TITLE
Add cache clear command

### PR DESCRIPTION
**Pull request by saraiva132**
sc-4774

### CODE REVIEW

File: src/cli/args.rs [hunk 1]
```diff
@@ -58,4 +58,7 @@ pub enum Command {
     },
     /// Cleans generated proto sources and lock file
     Clean,
+    /// Clears cached dependencies.
+    /// This will remove all cached dependencies and metadata hence making the next fetch operation slower.
+    CacheClear
```
Comment (zhrebicek) [inline src/cli/args.rs:63]: I would name it `ClearCache` just to be consistent with rest of the commands :)

Comment (saraiva132) [reply src/cli/args.rs]: Oki, agreed :) Done

File: src/cli/command_handlers.rs [hunk 2]
```diff
@@ -130,6 +130,20 @@ pub fn do_clean(lockfile_path: &Path, proto_output_directory: &Path) -> Result<(
...
 }
 
+pub fn do_cache_clear(cache: &ProtofetchGitCache,) -> Result<(), Box<dyn Error>> {
+    if cache.location.exists() {
+        info!(
+            "Clearing protofetch repository cache {}.",
+            &cache.location.to_string_lossy()
+        );
+        //TODO: understand whether there is a safer way to do this.
+        std::fs::remove_dir_all(&cache.location)?;
```
Comment (zhrebicek) [inline src/cli/command_handlers.rs:140]: Based on docs seems a way to go for me, so you can delete the comment.

**Review by zhrebicek — APPROVED** [review]
👍

**Pull request merged**